### PR TITLE
Ukraine statement

### PR DESCRIPTION
**Pull request by SeanKilleen**
Per discussion via e-mail amongst the core team.

### CODE REVIEW

**Comment by SeanKilleen** [comment]
Requesting official PR review from all involved in the discussion -- @OsirisTerje , @CharliePoole , @rprouse , @jnm2, @mikkelbu.

(Note that @jnm2 has indicated his agreement via e-mail previously, so if no response is received, we'll consider it approved by him.)

File: _posts/2022-02-26-ukraine-statement.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+We, like so many of our users and contributors, will continually be seeking ways to stand in solidarity with the Ukrainian people. We encourage you to donate to the [Ukrainian Red Cross](https://donate.redcrossredcrescent.org/), something each of us has done. You may also consider other donations at the [Stand with Ukraine page provided by the Ukrainian government.](https://ukraine.ua/news/stand-with-ukraine/)
+
+May Ukraine retain its sovereignty and regain its security, and may the Ukrainian people know peace as soon as possible.
+
+Signed,
+
+NUnit Core Team (in alphabetic order of last name)
+
+* Mikkel Bundgaard (TODO: Only with Mikkel's express approval)
```
Comment (mikkelbu) [inline _posts/2022-02-26-ukraine-statement.md:20]:
```suggestion
* Mikkel Bundgaard
```

**Review by mikkelbu — APPROVED** [review]

**Review by rprouse — APPROVED** [review]
Very well written, thank you @SeanKilleen

**Comment by Dreamescaper** [comment]
Thanks a lot, guys!
That matters a lot!

**Pull request merged**